### ISSUE TITLE
Remind me what's in it for us...?

### ISSUE THREAD

**Issue by eyalroz** [wesnoth/wesnoth#4866]
Wesnoth 1.15.3
Rise of Wesnoth
Scenario 5: Oldwood

> Prince Haldric: Lady, why would you let those undead monsters lead your people?
> 
> Lady Outlaw: Haldric, we are an ancient people who lived in a land filled with all manner of man and beast. We needed vision that extended beyond the meager span of human years. The art of Necromancy allowed our best and brightest to live forever, and our worst got to serve as mindless slaves.
> 
> Lady Outlaw: Other than losing our war back west, then our refugee war to your people, and this orc thing, it wasn't such a bad deal. 

Yeah, so, other than everything going horribly for them since forever -  being killed and turned into slaves for the undead was a swell "deal"? Seriously?

**Comment by CelticMinstrel** [comment]
I agree this is pretty questionable from a logical standpoint… perhaps it at least shouldn't be the "worst" that "serve as mindless slaves" since that suggests they're intentionally killing people to turn them into zombies, but rather the "unlucky" or something similar, implying that it's just those unlucky enough to die young.

**Comment by eyalroz** [comment]
@CelticMinstrel : That's still horrible...

There has to be some sort of clear benefit for the Wesfolk that Jenesse describes. Or some obstacle they were able to overcome, or danger they were able to avert. Either that or she bows to ages-old tradition without making a utility-based argument.

**Comment by ln-zookeeper** [comment]
Seems like a clear benefit to me if you can have your best and brightest kings live and rule forever instead of your kings constantly changing due to them dying (and inevitably ending up with really bad ones), and instead of simply executing or imprisoning your murderers and traitors, you get to convert them into untiring and uncomplaining workers who don't even demand pay.

How is it implausible for someone to consider than a good deal for their kingdom?

**Comment by eyalroz** [comment]
> Seems like a clear benefit to me if you can have your best and brightest kings live and rule forever instead of your kings constantly changing due to them dying

On the contrary. That means that new generations can never aspire to anything other than subservience. Not to mention the fact that once you're a Lich, your interests are no longer those of the nobility, nor even those of the living. Even ignoring Green Isle / Wesnothi morality, the living fear and dislike the dying and dead flesh and body, and more so the undead - on a very fundamental level, almost independently of cultural background.

Now, if Jenesse could have said "the Lich-Lords have saved us from calamities X Y and Z", then maybe, but like I said - the "rule of the best and the brightest" was apparently never anything other than daily terror, fear and exploitation on the micro level and utter failure and defeat on the macro level.

**Comment by ln-zookeeper** [comment]
> > Seems like a clear benefit to me if you can have your best and brightest kings live and rule forever instead of your kings constantly changing due to them dying
> 
> On the contrary. That means that new generations can never aspire to anything other than subservience.

And there's not really anything new or weird about people being happy with subservience if they feel it keeps them safe and gives them some kind of purpose, is there? You might be ruled by an undead monster, but at least it's an intelligent and competent one that on some level takes care of your people and keeps the even worse monsters from eating you. Jessene clearly isn't an idealist.

> Not to mention the fact that once you're a Lich, your interests are no longer those of the nobility, nor even those of the living.

For sure, they know that **now** after how it all turned out. She's saying, in past tense, that it wasn't such a bad thing... except for the bad things that consequently happened.

> Now, if Jenesse could have said "the Lich-Lords have saved us from calamities X Y and Z", then maybe, but like I said - the "rule of the best and the brightest" was apparently never anything other than daily terror, fear and exploitation on the micro level and utter failure and defeat on the macro level.

I think your suggestion is very much implied by what she's saying. She's saying their homeland was a rough place and they needed immortal strong leaders in order to survive long-term. The concept is straightforward and believable, so specifics of the dangers of their homeland don't really matter.

**Comment by eyalroz** [comment]
> And there's not really anything new or weird about people being happy with subservience if they feel it keeps them safe and gives them some kind of purpose, is there? 

That may be true for the general populace, not for the nobility. For long-standing empires, nobody refers to the ruling family as having been installed by some sort of a "deal". It's either "can't possibly be otherwise, never though of anything else" or some awesome powers they have as individuals or a line, or failing that - "The gods ordain it".

>  For sure, they know that now after how it all turned out. She's saying, in past tense, that it wasn't such a bad thing... except for the bad things that consequently happened.

It was in itself a bad thing. The question is - what was the offset?

> I think your suggestion is very much implied by what she's saying. She's saying their homeland was a rough place and they needed immortal strong leaders in order to survive long-term. The concept is straightforward and believable, so specifics of the dangers of their homeland don't really matter.

But she's not saying that. The strong immortal leaders simply failed. Was there anything which they did _not_ fail in? Did they ever _do_ anything which benefited the Wesfolk?

**Comment by CelticMinstrel** [comment]
> That's still horrible...

If it's not intentionally killing people to zombify them, then it's far _less_ horrible than the original comment implied.

> Seems like a clear benefit to me if you can have your best and brightest kings live and rule forever instead of your kings constantly changing due to them dying (and inevitably ending up with really bad ones)

Yeah, it _sounds_ good… until you remember that people change over time, and someone who's the best and brightest could later turn into a terrible tyrant.

**Comment by ln-zookeeper** [comment]
> > Seems like a clear benefit to me if you can have your best and brightest kings live and rule forever instead of your kings constantly changing due to them dying (and inevitably ending up with really bad ones)
> 
> Yeah, it _sounds_ good… until you remember that people change over time, and someone who's the best and brightest could later turn into a terrible tyrant.

Yes that's kind of the point Haldric makes in literally the very next line of dialogue. What's the problem with it? Is _"turning your best and brightest leaders into undead might actually backfire eventually"_ a too naive and simplistic narrative for a backstory? Or is it simply unbelievable that in a universe with liches and necromancers around every other corner, it might once, a long time ago, been considered an idea worth trying out in some distant culture on another continent?

**Comment by eyalroz** [comment]
> Overall, I don't understand what the problem here is. 

1. That Jenesse **doesn't indicate even one single positive aspect of the Lich-Lords' rule.**  The "best and brightest" ruling forever is not such a positive aspect, since those "best and brightest" have never done anything positive, they've only brought defeat and ruin.
2. That Jenesse is treating the fundamental structure of her society as a sort of a "deal". Societies don't work that way (even in a "magical" world).

**Comment by ln-zookeeper** [comment]
> > Overall, I don't understand what the problem here is.
> 
>1. That Jenesse **doesn't indicate even one single positive aspect of the Lich-Lords' rule.**  The "best and brightest" ruling forever is not such a positive aspect, since those "best and brightest" have never done anything positive, they've only brought defeat and ruin.

I simply don't see how that's a problem. She doesn't need to explain the positive aspects, it's not relevant to the TRoW story. She says it wasn't all bad, and whether that means she thinks they had really nice cobblestone roads or whether she thinks less people were eaten by giant scorpions is what's left up to your imagination. No story works without some cooperation from the audience, and to me it sounds like you're actively refusing using your imagination to fill this one unexplained detail in the backstory even when there's readily available explanations to pick from (the choice of which doesn't impact the story).

>2. That Jenesse is treating the fundamental structure of her society as a sort of a "deal". Societies don't work that way (even in a "magical" world).

I don't see how that's true. Even in the real world, surely nobility and royalty have tended to have a mutually beneficial relationship and plenty of monarchs and lineages have been deposed by nobility who no longer considered the ruler to uphold their interests. Do you consider for example GoT to be unbelievable because plenty of characters in-universe understand it's all just a big machiavellian power struggle and that people only sit on the throne thanks to "deals" and not divine right?

**Comment by eyalroz** [comment]
>  She doesn't need to explain the positive aspects, it's not relevant to the TRoW story. 

It _is_ relevant, because when the background story seems flimsy, or not believable enough, this detracts from players' identification with the character. She says it wasn't all bad - as a conclusion to explaining how it _was_ all bad. If the allegiance is justified in terms of a deal, then there needs to be coherent justification; Jenesse needs to believe that what she's telling Haldric makes sense and doesn't make her look ridiculous. If you don't want to tell the cobblestones and scorpions story - or more significantly expand the history/mythology of Wesnoth, then it's better to chalk it up to "This has been our way for generations" or something along those lines. Or - just make it a comment by Haldric which is brushed off by Jenesse.

About the real-world feudalism - you're arguing against your own point. For feudal nobility, the ruling line is part of their own social class, and occasionally gets replaced; but with the Lich-Lords - there are no checks by the nobility and the possibility of deposition. As for GoT - characters don't treat ages-old monarchy of foreigners as a "deal", except possibly dispassionate and disloyal cynics (e.g. Varis or Littlefinger types).

**Comment by CelticMinstrel** [comment]
> Yes that's kind of the point Haldric makes in literally the very next line of dialogue.

Yeah, okay, that's fine then. I'm not fully versed in the context of this text, just commenting on what the original poster said.

**Comment by Ja-MiT** [comment]
I've been viewing those lines as more humorous, and a bit more tongue-in-cheek, than this discussion indicates. My interpretation had been along the lines of *"It's been great. For centuries. Well, not so great when we finally lost a war. And then kept losing. Um... Can we maybe ignore the past few decades?"* Sure, the time frames ("centuries" and "decades") involve significant inference on my part, but I thought it was the logical conclusion, especially given the lack of specific benefits. That is, there must have been so many benefits over such a long period of time that picking a few to mention would be a disservice to the others. The lack of details speaks volumes. Then again, that is my personal read and not necessarily shared by others.

On a more objective note, the debate over "deal" leads me to believe there is a conflict of definitions going on.  
**Plausible writer's definition:** "deal" meaning "a situation"  
**Apparent reader's definition:** "deal" meaning "an agreement"  
(See [Wiktionary](https://en.wiktionary.org/wiki/deal#Noun_2) for a reference, if you like. These definitions are numbers 6 and 5 of the noun definitions under "Etymology 2".) The meaning of "deal" meaning "situation" is informal, though, and perhaps feels more modern than what Wesnoth typically portrays. Maybe change "deal" to "situation" or "state of affairs" or some other synonym?

**Comment by eyalroz** [comment]
@Ja-MiT : You make an important point about the dialog seeming tongue-in-cheek. I also felt like that. But - this shouldn't happen. Jenesse is supposed to be a tragic character, as is the situation of the Wesfolk. It's disrespectful to assign her humorous dialog on such a subject, in this context - especially when the actual answer to the question is not known to the player.

**Comment by 123chess456** [comment]
I think she's being sarcastic.

**Comment by eyalroz** [comment]
@123chess456 : That's the thing - she isn't. See what @ln-zookeeper writes. She's actually serious... which is preposterous.

**Comment by CelticMinstrel** [comment]
It doesn't seem so totally preposterous. She's implying that there used to be good sides to it, which might have been short-sighted (if they'd given it more thought they could've predicted it would go badly) but probably isn't technically false.

**Comment by eyalroz** [comment]
@CelticMinstrel : What's preposterous is her being flippant, when talking to the leader of refugees who have lost their homeland, their way of life, and many of their friends and family members -  in no small part due to her people's and her own personal actions.

**Comment by ln-zookeeper** [comment]
As said, the campaign has a lot of tongue-in-cheek in it, this being one (minor) example of that. Whether one finds a humorous tone appropriate or not in any given context is, of course, a matter of taste. For example I find that sort of levity to be one of the most appealing parts of the campaign's dialogue.

**Comment by Wedge009** [comment]
As others have already mentioned, I took this 'deal'/situation that the Wesfolk find themselves in, according to Jessene, was a benefit because of how terrible the place where they came from is supposed to be. To me, 'all manner of man and beast' is supposed to be a terrible experience, nothing at all like what eventually becomes the civilisation of Wesnoth or even the various smaller kingdoms of the Green Isle were before the invasion.

Looking at the bigger picture, the plot simply needs a reason for the Haldric's people - and the Wesfolk too - to flee and eventually establish the Kingdom of Wesnoth - it's what the campaign's name is all about after all. The Wesfolk's reason was that their lich-lords betrayed them in unleashing the orcs upon the Isle. And as a player, what happened between the Wesfolk and the lich-lords in times past doesn't really matter so much - all they need to know is that they are now in this adventure along with Haldric and his people.

If all this feels unreasonable, consider the other end of the Wesnoth timeline, with the radical change in direction that Under the Burning Suns takes compared with the other campaigns. There may be things there that don't make sense there either, like the hubris and conjuring of the second sun, etc, but ultimately this is a game and the story supports the game. Like a lot of movies, some things may happen that don't necessarily make sense, but usually audiences don't let that get in the way of their enjoyment.

Finally, I think this kind of discussion is best suited for the forums, where I understand the community tears apart story and dialogue in order to rewrite and improve. And @nemaara may have something in mind for The Rise of Wesnoth in the near future, I don't know. So like #4865, I'm inclined to close this because this is more of a non-technical discussion on the writing of a particular part of a campaign (which carries across just fine for me, and many others by the looks of it). And it's been nearly 18 months since the last response, I don't really see any benefit to keeping this open.

**Comment by nemaara** [comment]
I don't have anything planned for TRoW. I don't mind if you close it, I tend to look at these things when doing writing touchups too.

**Comment by eyalroz** [comment]
I believe the most important phrase in your comment is:

> **the plot simply needs a reason for the Haldric's people**

https://i.imgflip.com/5n34p8.jpg

We are talking about the very foundations of Wesnothi civilization here, my friend! There is no "simply" here... it's not just some random blurb on some scenario, like why we're attacking the Saurians from this direction rather than that :-(

> I took this 'deal'/situation that the Wesfolk find themselves in, according to Jessene, was a benefit because of how terrible the place where they came from is supposed to be.

If the dialog were to say this explicitly, that would help.

> To me, 'all manner of man and beast' is supposed to be a terrible experience

Variety doesn't sound so terrible. Also, this doesn't immediate relate to the following sentence she says about "needing vision". If she would say they were being _prayed on_ by bandits and beasts, with the necromancers' rule staving off that threat or stabilizing things, that would be plausible even if not very detailed.

Also remember that some commenters here took her words as a _joke_, as something said tongue-in-cheek, "we needed them to rule us because of, uh, reasons or whatever, and they got to make us slaves. It was the deal of the century."

> nothing at all like what eventually becomes the civilisation of Wesnoth

1. I dunno, Wesnoth can be described as a land with all manner of man and beast.
2. Regardless, that's not the right comparison. That wasn't the deal. The necromancers weren't allowed to take power to lead the Wesfolk away from their lands to someplace else - the deal was for them to rule the Wesfolk *in-place*. The necromancers didn't offer the Wesfolk to go anywhere, they ran to _escape_ the deal. So, again, one is left wondering what kind of deal it was.

> If all this feels unreasonable, consider the other end of the Wesnoth timeline, with the radical change in direction that Under the Burning Suns takes compared with the other campaigns. 

I played the (recently revamped I understand) UtBS. It's true that that's a whole lot to unpack; and certainly some points are really not well explained (like how machines got into the mix). But - there is a _whole damn lot_ of explaining and back story. Imagine if the explanation you got for how the Quenoth Elves came to be how they are is:

> Character X: Hey Kaleh, I thought Elves were fair-skinned and lived in forests. What gives?
>
> Kaleh: Well, X, our people used to live in a great forest, but it was filled with all manner of man and beast. And we lacked vision! Then we were visited by these priestesses, and they offered us a deal. The deal was that we would leave our beautiful homelands and go rough it out in the desert, never to return. So, that happened.


Finally, about the technical nature: As it stands now, the dialog is inconsistent and sends conflicting messages. You understood one thing; I understood another thing; @Ja-MiT understood the whole thing to be tongue-in-cheek. IMHO this state of affairs constitutes a bug, rather than just something which may potentially be developed and rewritten as scenarios sometimes are. However - I'm open to the idea of starting a forum thread while this bug remains open for at least some sort of temporary fix (e.g. along the lines of your understanding and as I suggested above).

**Comment by Wedge009** [comment]
The point of that phrase is not the variety, it's an expression conveying the trials they faced before arriving on the Green Isle. I'd also argue that differing interpretations in literature is a feature rather than a bug given how good storytelling is received in other contexts. But I don't plan to spend my valuable time continuing with this, and it seems you've completely missed the point that this isn't the most constructive place to be having such a discussion. I think the level of attention you're paying to the detail is admirable, but not necessarily constructive when it doesn't appear that many others share a similar view. I invite you to have your say on the forums where you can have this kind of conversation with a wider audience and among fellow players, eg https://forums.wesnoth.org//viewtopic.php?t=1510 and develop a way forward there. Here, you're mainly going to get opinions and thoughts from the perspective of developers, even if some are also players as well.

---

After discussing with @nemaara I concede there is room for improving the back-story with relation to why the Wesfolk would suffer living under the lich-lords. However, we did agree that the GitHub issue tracker is probably not the best place to discuss such nuance and also considered that the history of the Wesfolk was deliberately left vague in the campaign. It also takes considerable skill to write with the dry wit and dead-pan humour of a character like Jessene, and that's not going to be something that's easily handled in an issue like this. (Contrast with #4401, for example, that has both a consensus as well as more solid ideas on how to improve things.)

So only with respect to having a reminder to consider this improvement, I've renamed the issue to reflect what we've discussed here. I still advise further discussion to be made in the forums, unless there are concrete suggestions on what to change in the text.

> [Yumi] the way it's presented in TRoW doesn't entirely make logical sense, jessene as a character is practical to a point, but clearly still values the life of regular people and not just the "best and brightest", I'd argue that she's even more caring in a way than Haldric
[Yumi] and yet if we take her seriously when talking about the Wesfolk that's a contradiction with her character
[Yumi] unless one makes the argument that the lich lords betraying them was a wakeup call
[Yumi] in either case, I think the tone of those lines would need to be a little more ironic if they are to be taken as factual, as in "yea this happened, and yea this is the way we were (btw we obviously aren't like that now)"
[Yumi] or we modify the lichlords' origin story a little bit
[Yumi] I think that requires a bit of skill to do

**Comment by AI0867** [comment]
Regarding the "why they would suffer living under the lich-lords", this implies that the Lich-Lords are a sort of foreign overlord that has subjugated the Wesfolk.

A scenario I find more plausible is the following:

1. During some war or calamity, one or more mages use necromancy to save the day.
2. Unlike Malin, these mages aren't exiled or executed.
3. Due to further helpful acts of these mages, necromancy becomes more accepted.
4. A popular ruler happens to be a mage, and turns himself into a lich, allowing his rule to continue indefinitely.
5. Other highly capable mages serving the ruler are also elevated to lichdom, so their abilities will not be lost to death.
6. Perhaps the goals of the ruling class eventually diverge from that of the ruled class.

**Comment by eyalroz** [comment]
@AI0867 : Do you think we should move this discussion to the forum perhaps?

Anyway, I think the story you sketch out is at the very least far superior to what we have now. However, your final point can't just be a "perhaps": Before the campaign starts, significant numbers of Wesfolk have decided to flee their lands, over a prolonged period of time, enough to harass the Green Islanders militarily and not just a helpless refugees. So, things must have been pretty dire for quite a while already, and this must be explained at least from the point of view of the refugees.

**Comment by stevecotton** [comment]
The original post's quote stops half-way through a line of dialogue. Jessene's line in full is "Other than losing our war in the far West, then our refugee war to your people, and this orc thing, it wasn’t such a bad deal. Lich-Lord Lenvan was one of our greatest leaders. **He led our escape to this land.** Albeit, after being down in that hole for so long I don’t imagine he’s too happy."

The bit in bold seems to resolve many of the questions asked both here and in the forum thread.

**Comment by eyalroz** [comment]
@stevecotton :  Hmm. You're right. I've forgotten about that since I opened the issue last year...

But it actually keeps open as many questions as it might be resolving:

* If the Wesfolk arriving on the Green Island were an organized escape led by Lenvan, why are they described as "the vanquished refugees of some war"? Did Lenvan's faction fight a civil war, lose and leave, or just leave?
* It's not clear who were the belligerents in the "war in the far West"; but it seems that it was not the (potential) civil war between the Lenvan  faction and the majority of the Lich-lords' followers - because it's being discussed in context of evaluating the overall 'Lich-lord deal'.
* What did the Lich-lords need the Orcs for, and at what point? Was it for the war in the far west? Was it part of inter-Lich-lord struggles? Was it to suppress the Humans further?
* What was the upside of the deal with Lich-lords in the first place - and why did Lenvan and/or the humans under him decide to give up on it?
* What sets Lenvan apart from the other Lich-lords?
